<?xml version="1.0" encoding="UTF-8"?>
<!--
This file is a part of __PROGRAM_NAME__ __PROGRAM_VERSION__

This file installs __PROGRAM_NAME__ in the operating system, cleans
temporary files and directory in the project.

__PROGRAM_COPYRIGHT__ __PROGRAM_AUTHOR__ __PROGRAM_AUTHOR_EMAIL__

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program.  If not, see <http://www.gnu.org/licenses/>.
-->
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>805</x>
      <y>270</y>
      <width>86</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutContolButtons">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonControlButtonsScan">
       <property name="text">
        <string>Scan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonControlButtonsCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonControlButtonsQuit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxPorts">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>331</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Ports</string>
    </property>
    <widget class="QLabel" name="labelPortsFirst">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>First</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPortsLast">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Last</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxPortsFirst">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPortsClear">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>84</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxPortsLast">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPortsCopy">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxPortsEnabled">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>93</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enabled</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxIPRange">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>IP Range</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>272</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutIPRangeFirst">
      <item>
       <widget class="QSpinBox" name="spinBoxIPRangeFirst1">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxIPRangeFirst2">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxIPRangeFirst3">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxIPRangeFirst4">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>272</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutIPRangeLast">
      <item>
       <widget class="QSpinBox" name="spinBoxIPRangeLast1">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxIPRangeLast2">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxIPRangeLast3">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxIPRangeLast4">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="labelIPRangeFirst">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>First</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelIPRangeLast">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Last</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonIPRangeCopy">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonIPRangeClear">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>170</y>
       <width>84</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>781</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
